Add draft of 2021-09-13 leadership meeting minutes (#135)
carpentries/trainers: leadership_minutes/2021-09-13-leadership-minutes.md

    - Sarah: Timing questions?
    - Karen: If you've been active for the past year, you are renewed for the following year
    - Amanda: I've read this a couple different ways. Does this apply to me (as someone who has done stuff) or someone who hasn't done things and want to fix it.
    - Karen: Both.
    - Amanda: Is there a column C for people who don't do either thing?
    - Karen: Yes. We actually have procedures for that already, but we should make sure they are aligned. [Trainer Alumni Status](https://docs.carpentries.org/topic_folders/instructor_training/duties_agreement.html#trainer-alumni-status)
    - Amanda: I like how simple it is. How long is expired?
    - Arin: This signals that there is a system that tracks. So compared with what we have now, does it create more overhead?
    - Karen: This is where the survey comes in that I proposed last time. We ask them what they have been doing.
    - Arin: Should we not remind people or warn them if they are not meeting expectations?

>     - Sarah: My gut response is that people need to be adults and keep track of their own. If it is easy to automate, OK, but otherwise I don't think its a good use of core team time.

Reviewer: ```suggestion
    - Sarah: My gut response is that people are adults and can keep track of their own. If it is easy to automate, OK, but otherwise I don't think its a good use of core team time.
```
